torchgeo/datasets/openbuildings.py

```diff
@@ -0,0 +1,462 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Open Buildings datasets."""
+
+import glob
+import json
+import os
+import sys
+from typing import Any, Callable, Dict, List, Optional
+
+import fiona
+import fiona.transform
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
This should be a lazy import, we don't yet have a required dependency on pandas. This is causing the RTD tests to fail. Not sure why the Windows tests are failing though. See `idtrees.py` for an example where we do this. The tests should also skip if pandas isn't found, see `test_idtrees.py`